src/main/java/mate/model/Car.java

```diff
@@ -9,6 +9,16 @@ public class Car {
     private Manufacturer manufacturer;
     private List<Driver> drivers;
 
+    public Car(String model, Manufacturer manufacturer, List<Driver> drivers) {
+        this.model = model;
+        this.manufacturer = manufacturer;
+        this.drivers = drivers;
+    }
+
+    public Car() {
+
```

Comment:
```suggestion
```

Reply:
It's maven rule to have one empty line between methods

Reply:
probably @Gamemod13 meant an empty line within a constructor